pkg/converter/muxer.go

```diff
@@ -75,20 +76,37 @@ func FromContentType(ctype string, contents []byte) *Muxer {
 }
 
 // Muxer converts from an object's serialized format to an actual instance of
-// the object.
+// the object. By default, the Muxer does not allow unknown fields.
 type Muxer struct {
-	data   []byte
-	format ContentType
+	data               []byte
+	format             ContentType
+	allowUnknownFields bool
+}
+
+// AllowUnknownFields indicates whether to allowUnknownFields
+func (m *Muxer) AllowUnknownFields(allow bool) *Muxer {
+	m.allowUnknownFields = allow
+	return m
 }
 
 func (m *Muxer) mux(f interface{}) error {
 	switch m.format {
```

Comment:
it doesn't matter either way, right, since it's private? But perhaps I'm missing something?

So you'd suggest:

```go
func mux(f interface{}, data []byte, format ContentFormat, allowUnknownFields bool) ?
```

Reply:
It does not matter much because it's private to this package, yes. 

What you wrote is what I am saying.   Also probably rename the generic interface value because it's not obvious what it does without looking into the implementation.  By docs says the value it stored at f.  It would be nice to have a convention to dictate when functions have side effects.  Is f and input or output?   We don't have const parameters, but we can create nicities like that by convention.